extensions/parquet/table/build.gradle

```diff
@@ -52,7 +52,6 @@ dependencies {
     Classpaths.inheritSlf4j(project, 'slf4j-simple', 'testRuntimeOnly')
 
     runtimeOnly project(':extensions-trackedfile')
-    testImplementation project(':extensions-s3')
```

Comment:
This can be changed to 

```gradle
testImplementation TestTools.projectDependency(project, 'extensions-s3')
```

to pull the testlib into parquet tests.